### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by BensonCasey7**

### CODE REVIEW

**Review by floatkeera — DISMISSED** [review]
jklhklhlklkl

File: README.md [hunk 1]
```diff
@@ -5,4 +5,4 @@ wthr. utilizes the OpenWeatherMap API for real-time weather information, as well
 This test app can be viewed at https://floatwthr.firebaseapp.com/
 
 ## Dependencies
-- npm
+- npmf
```
Comment (floatkeera) [inline README.md:8]: Please restore.

**Review by floatkeera — DISMISSED** [review]
Over all its good but....

**Review by floatkeera — DISMISSED** [review]
Its not very good

**Review by floatkeera — COMMENTED** [review]
hi

**Comment by floatkeera** [comment]
LOL

**Review by floatkeera — APPROVED** [review]
yay

File: README.md (hunk 1 above)
Comment (floatkeera) [reply README.md:8]: hi

**Comment by BensonCasey7** [comment]
@floatkeera testing

**Pull request merged**